utils/emailTemplate.js

```diff
@@ -0,0 +1,22 @@
+exports.accountVerification = function(name, token) {
+  let content = `Hi ${name},<br/><br/>
+  Click the <a href='http://localhost:3080/api/ngo/verify/email/${token}'>verification link</a> to activate your account.`
```

Comment:
The url has to be moved to .env and loaded from there.
You will also need a record id or base64'd user email as part of the request. Relying on just the token to fetch and update record may fail.
/verify/<base64_email or base64(mongo-obj)/<token>

Reply:
Will take it up

Reply:
Moved URL to .env
Update the API to get the email in base64 format